src/app/_components/MarkdownPage.tsx

```diff
@@ -0,0 +1,33 @@
+import type { WithContext, Thing } from 'schema-dts'
+
+import type { PageAttributes } from '@/types/attributesTypes'
+
+import type { SeoMetadata } from '@/schemas/SeoMetadataSchema'
+
+import { MarkdownContent } from '@/components/MarkdownContent'
+import { PageHeader } from '@/components/PageHeader'
+import { StructuredDataScript } from '@/components/StructuredDataScript'
+
+type MarkdownPageProps = {
+  attributes: PageAttributes
+  body: string
+  generateStructuredData: (seo: SeoMetadata) => WithContext<Thing>
+}
+
+export function MarkdownPage({
+  attributes,
+  body,
+  generateStructuredData,
+}: MarkdownPageProps) {
+  const { header, seo } = attributes
+
+  return (
+    <article>
+      <StructuredDataScript structuredData={generateStructuredData(seo)} />
+      <header className="mb-6">
+        <PageHeader.Title>{header.title}</PageHeader.Title>
```

Comment:
Not sure if it’s a major issue: Should we limit the page title width to match the content below?

On `/employee-privacy-policy`, the header is wider than the body content.

Reply:
Yes! Good catch. Thank you. I added the `prose` class to `article`.